tracer/test/Datadog.Trace.Tests/Configuration/OverrideErrorLogHelper.cs

```diff
@@ -0,0 +1,53 @@
+﻿// <copyright file="OverrideErrorLogHelper.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System.Collections.Generic;
+using System.Linq;
+using Datadog.Trace.Configuration.ConfigurationSources.Telemetry;
+using Datadog.Trace.Logging;
+using Datadog.Trace.Telemetry;
+using Datadog.Trace.Telemetry.Metrics;
+using FluentAssertions;
+using MetricDefinition = (Datadog.Trace.Telemetry.Metrics.Count? Metric, string OtelSetting, string DdSetting);
+
+namespace Datadog.Trace.Tests.Configuration;
+
+internal static class OverrideErrorLogHelper
```

Comment:
Should we call this `OverrideErrorLogExtensions` to follow our general conventions (and make it more obvious to readers?)

Reply:
We have a lot of `Helper` conventions too 😉 I'm not passionate about it at all, so sure 😄

Reply:
I think usually our `Helper` types have static methods that are _not_ extensions? Not really an important distinction or anything. Just nit picking 😅